### ISSUE TITLE
Program does not work well with Linux screenreader Orca

### ISSUE THREAD

**Issue by pixelglade** [babluboy/bookworm#399]
I was trying to use Orca with Bookworm but encountered issues with using it such as the inability to tab into the main text area and navigate by sentences with the keyboard arrow keys. If using tab you can jump to the next link but are stuck if you want to try leave the link and read the surrounding text. If using the mouse to force Orca to read the selected text is will read the selected text but using the bumper keys on the Orbit 20 will scroll the text until it reaches the end of the current paragraph and then you cannot go any further unless you click somewhere else on the page. Entering full screen mode does not make a difference.

I'm not sure if the development team has done any testing with Orca, if there are any known work arounds I would appreciate some advice. Otherwise, it might be helpful to do some testing with Orca to try make the page more accessible to those using keyboard, screenreaders, or refreshable braille displays.

**Comment by babluboy** [comment]
Many thanks for your interest in bookworm and for testing it with Orca. I'm the sole developer of Bookworm and I was not aware of Orca and its capabilities. I will keep this issue open to remind me to check out Orca and screen readers in general so that I can create whatever compatibility is possible with bookworm

**Comment by pixelglade** [comment]
I appreciate your willingness to the test the program with Orca. 

I'm not sure if you're aware of this but there is a Windows-based ebook reader which is accessible (designed for the vision impaired) and shares the same name as your ebook reader: [Bookworm (by github user blindpandas)](https://github.com/blindpandas/bookworm). Before using Linux I used Bookworm on Windows to connect with my braille display. If you have access to Windows I would recommend checking out how the program as far as testing how it works with a screenreader goes (NVDA is a free screen reader for Windows). 

Someone on YouTube ([How to Use Bookworm](https://www.youtube.com/watch?v=NllQpkISObM)) has made a video interacting with the Windows Bookworm with a screenreader. The part that relates to what I mentioned is around 4 minutes in but it involves the ability to seamlessly move between the main document window, the heading overview, and the toolbar.

Here is the link to the [Orca screen reader](https://help.gnome.org/users/orca/stable/) if you wanted to try it.

And the W3C has made a [User Agent Accessibility Guidelines (UAAG)](https://www.w3.org/WAI/standards-guidelines/uaag/) which include applications like readers. It has a [Reference for its 2.0 standards](https://www.w3.org/TR/UAAG20-Reference/), and a section specifically for how to make programs work with [Accessible Technology](https://www.w3.org/TR/UAAG20-Reference/#gl-AT-access) and what that should look like. This might help you get started. 🙂
